xarray/core/duck_array_ops.py

```diff
@@ -199,6 +200,17 @@ def allclose_or_equiv(arr1, arr2, rtol=1e-5, atol=1e-8):
     """
     arr1 = asarray(arr1)
     arr2 = asarray(arr2)
+    # TODO: remove after we require dask > 2.9.1
```

Comment:
let's wait to check the tests but this bit should go under `if lazy_equiv is None:`

Reply:
true. There was also a mistake in the way I checked the version